src/components/04-tables/tables.jsx

```diff
@@ -0,0 +1,96 @@
+/**
+ * JSX Tables Component Attributes
+ * @function Tables
+ * @param {Object} config - The configuration object for table attributes
+ * @property {string} config.name  - The name of the table.
+ * @property {string} config.class - The classes inherited from the USWDS CSS files. The default class is `usa-table` and additions can be appended.
+ * @property {Array} config.children - When this template is used to make a react component @babel/core will change "config" to "props" so that children components can be accessed.
+ * @property {string} config.id    - The ID of the table
+ * @example
+ *   const config = {
+ *     name: "Table",
+ *     class: "usa-table--borderless",
+ *     id: "813"
+ *   };
+ *   Tables(config)
+ * @returns {HTMLTableElement} The HTML Table
+ */
+exports.Tables = function (config) {
+  return (
+    <table id={`${config.tableID}`} class={`usa-table ${config.class}`}>
+      <caption>{config.caption}</caption>
+      {config.children}
+    </table>
+  );
+};
+
+/**
+ * @function TableHeader - function that returns HTML code for a table header
+ * @param {Object} config - configuration object for tables
+ * @property {Array} config.header - The array of data for the column header.
+ * @example
+ *  const config = {
+ *   header: [
+ *       "Document title",
+ *       "Description",
+ *       "Year"
+ *     ]
+ * }
+ * TableHeader(config);
+ * @returns {HTMLTableHeaderCellElement} - The HTML for the table header
+ */
+exports.TableHeader = function (config) {
+  let tableHeader = [];
+
+  for (let i = 0; i < config.header.length; i++) {
+    tableHeader.push(`<th scope='col'>${config.header[i]}</th>`);
```

Comment:
non-blocker: is there a use case where a developer might want to pass a class to a `<th>`? If so, it might be useful to add make each header an object: `<th ${config.header[i].class} scope='col'>${config.header[i].text}</th>`. However there are complexity downsides to this which would need to be weighed

Reply:
I took your advice and did exactly this. Each header is an object with **class** and **text** properties.